lib/errors.js

```diff
@@ -15,7 +15,10 @@ module.exports.BODY_LINES_MAX_LENGTH = Error('No line in the body should exceed
 
 /**
 * @name no-tag-in-body
-* @description Commit body should not contain footer tags
+* @description 
+* Commit body should not contain footer tags.
+* This error might also be caused by a footer containing anything other than a valid tag.
```

Comment:
I feel this wording is a bit too generic, its not like any invalid footer may cause this, but only if its the first footer between the body and the others (because in that case its interpreted as part of the body and this kicks in).... cant really think of a good way to phrase it though. Definitely we should link to pretty-tags from here

Reply:
I agree that it is somewhat generic, but I think it is sufficient as a guide on where to look if you are having trouble with a commit. Another wording might be: "**This error might also be caused if the first line of the footer is not a tag. Refer to....**". What do you think?

I will add a markdown link to pretty-tags.

Reply:
@sradevski That sounds better, thanks :)